.github/workflows/performance-regression.yml

```diff
@@ -1,152 +1,140 @@
-name: Performance Regression Detector
+name: Performance Regression
 
 on:
   pull_request:
-    branches: [ master ]
+    paths-ignore:
+      # Workflow will not run if all changes match these paths
+      - '**/*.md'
```

Comment:
This is the replacement for the PR-content-check, correct? Can we just opt-in `amazon/ion/**` rather than excluding just the markdown files?

Reply:
Yes, we can do that.